{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-0483", 
                "published": "2014-08-26T10:55:05.347-04:00", 
                "id": "CVE-2014-0483", 
                "modified": "2014-11-13T22:01:30.197-05:00", 
                "summary": "The administrative interface (contrib.admin) in Django before 1.4.14, 1.5.x before 1.5.9, 1.6.x before 1.6.6, and 1.7 before release candidate 3 does not check if a field represents a relationship between models, which allows remote authenticated users to obtain sensitive information via a to_field parameter in a popup action to an admin change form page, as demonstrated by a /admin/auth/user/?pop=1&t=password URI."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/264.html", 
                "id": "CWE-264", 
                "title": "Permissions, Privileges, and Access Controls"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/a:djangoproject:django:1.7:rc2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6:-"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:rc1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5:alpha"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.5"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.6"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6.3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.5"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6:beta2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6:beta4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6:beta1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6:beta3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.7:beta1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.10"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.11"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.13"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.7"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.8"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.8"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.9"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6.5"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.6"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.7"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.12"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.6.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5:beta"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.67425", 
                "id": 67425
            }
        ], 
        "IAVM": [], 
        "BID": [], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "https://github.com/django/django/commit/2b31342cdf14fc20e07c43d258f1e7334ad664a6", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "https://www.djangoproject.com/weblog/2014/aug/20/security/", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://www.debian.org/security/2014/dsa-3010", 
                    "vendor": "DEBIAN"
                }, 
                {
                    "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html", 
                    "vendor": "SUSE"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "2.9", 
                    "access complexity": "medium", 
                    "confidentiality impact": "partial", 
                    "authentication": "single_instance", 
                    "exploit": "6.8", 
                    "base": "3.5", 
                    "integrity impact": "none", 
                    "availability impact": "none", 
                    "access vector": "network"
                }
            ], 
            "severity": "Low", 
            "Top alert": false, 
            "Top vulnerable": false
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2014-0483"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:25935", 
                "title": "DSA-3010-1 python-django - security update", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:25935"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:26675", 
                "title": "USN-2347-1 -- python-django vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:26675"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "Debian Security Advisory DSA 3010-1 (python-django - security update)", 
                "file": "deb_3010.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "703010"
            }
        ], 
        "Nessus": [
            {
                "name": "Debian DLA-65-1 : python-django security update", 
                "file": "debian_DLA-65.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "82210"
            }, 
            {
                "name": "Debian DSA-3010-1 : python-django - security update", 
                "file": "debian_DSA-3010.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "77344"
            }, 
            {
                "name": "Fedora 20 : python-django14-1.4.16-1.fc20 (2014-15266)", 
                "file": "fedora_2014-15266.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "79645"
            }, 
            {
                "name": "Fedora 19 : python-django14-1.4.16-1.fc19 (2014-15307)", 
                "file": "fedora_2014-15307.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "79646"
            }, 
            {
                "name": "Fedora 20 : python-django-1.6.6-1.fc20 (2014-9771)", 
                "file": "fedora_2014-9771.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "77590"
            }, 
            {
                "name": "Fedora 20 : python-django14-1.4.14-1.fc20 (2014-9788)", 
                "file": "fedora_2014-9788.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "77591"
            }, 
            {
                "name": "Fedora 20 : python-django15-1.5.9-1.fc20 (2014-9866)", 
                "file": "fedora_2014-9866.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "77614"
            }, 
            {
                "name": "FreeBSD : django -- multiple vulnerabilities (3c5579f7-294a-11e4-99f6-00e0814cab4e)", 
                "file": "freebsd_pkg_3c5579f7294a11e499f600e0814cab4e.nasl", 
                "family": "FreeBSD Local Security Checks", 
                "id": "77315"
            }, 
            {
                "name": "GLSA-201412-22 : Django: Multiple vulnerabilities", 
                "file": "gentoo_GLSA-201412-22.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "79975"
            }, 
            {
                "name": "Mandriva Linux Security Advisory : python-django (MDVSA-2014:179)", 
                "file": "mandriva_MDVSA-2014-179.nasl", 
                "family": "Mandriva Local Security Checks", 
                "id": "77658"
            }, 
            {
                "name": "openSUSE Security Update : python-django (openSUSE-SU-2014:1132-1)", 
                "file": "openSUSE-2014-542.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "77718"
            }, 
            {
                "name": "Ubuntu 10.04 LTS / 12.04 LTS / 14.04 : python-django vulnerabilities (USN-2347-1)", 
                "file": "ubuntu_USN-2347-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "77725"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [
            {
                "url": "https://www.suse.com/security/cve/CVE-2014-0483.html", 
                "id": "openSUSE-SU-2014:1132"
            }
        ], 
        "Redhat": [], 
        "Gentoo": [], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [], 
        "Debian": [
            {
                "url": "https://security-tracker.debian.org/tracker/DSA-3010", 
                "id": "DSA-3010"
            }
        ], 
        "Vmware": []
    }
}